table.csv

```diff
@@ -482,6 +482,10 @@ skein1024-1000,                 multihash,      0xb3dd,         draft,
 skein1024-1008,                 multihash,      0xb3de,         draft,
 skein1024-1016,                 multihash,      0xb3df,         draft,
 skein1024-1024,                 multihash,      0xb3e0,         draft,
+xxh-32,                         multihash,      0xb3e1,         draft,
+xxh-64,                         multihash,      0xb3e2,         draft,
+xxh3-64,                        multihash,      0xb3e3,         draft,
+xxh3-128,                       multihash,      0xb3e4,         draft,
```

Comment:
```suggestion
xxh-32,                         hash,           0xb3e1,         draft,
xxh-64,                         hash,           0xb3e2,         draft,
xxh3-64,                        hash,           0xb3e3,         draft,
xxh3-128,                       hash,           0xb3e4,         draft,
```

as per https://github.com/multiformats/multihash/pull/156, if that's accepted

Reply:
I think also in the last column you should clarify that it's non-cryptographic, something like "non-cryptographic xxHash variant"